app/services/decidim/consultations/participation.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Consultations
+    # Computes the participation into a Conslutation after this Consultation is closed.
+    Participation = Struct.new(:consultation) do
+      # Returns a list of participants as an Array of Arrays.
+      # Thus, each user is represented by an Array of the following form:
+      # [
+      #   author.name,
+      #   author.email,
+      #   did_vote?,
+      #   delegated_vote?,
+      #   vote.created_at.rfc3339
+      # ]
+      #
+      def results
+        raise "Can not compute results on a Consultation that is not closed." unless consultation.finished?
+
+        org_users= Decidim::User.where(organization: consultation.organization).order(:name)
+
+        org_users.filter_map do |user|
+          user_votes = user_votes_on_current_consultation(user)
+          last_vote = user_votes.order(:created_at).last
+
+          next unless last_vote
+
+          [
+            user.name,
+            user.email,
+            last_vote.present?,
```

Comment:
Maybe this is always `true`? because if `last_vote` is not present then the next iteration is done on the `filter_map`.

can you check it?

Reply:
yes it is redundant, but is correct